src/Core/Content/Sitemap/Provider/ProductUrlProvider.php

```diff
@@ -96,11 +96,11 @@ public function getUrls(SalesChannelContext $context, int $limit, ?int $offset =
             $urls[] = $newUrl;
         }
 
+        /** @var list<string> */
```

Comment:
```suggestion
        /** @var list<string|int> */
```
array keys can be both string and integers

Reply:
The array is returned from the FetchModeHelper::keyPair($products); which always returns "array<string, string>"

Reply:
then why is this annotation needed at all? :thinking:

Reply:
We no longer need it, i removed it and it still works fine ^^